modules/vba_documents/lib/vba_documents/multipart_parser.rb

```diff
@@ -25,9 +25,8 @@ def self.parse_file(infile)
                 else
                   infile
                 end
-        validate_size(input)
```

Comment:
Removed this line since true "Empty payloads" won't reach the `MultipartParser` anymore.